heft-plugins/pre-compile-hardlink-or-copy-plugin/README.md

```diff
@@ -0,0 +1,39 @@
+# @rushstack/pre-compile-hardlink-or-copy-plugin
+
+This simple plugin provides a method for hardlinking or copying a file, or
+recursively hardlinking or copying files in a folder.
+
+To use this plugin, install it in your project:
+
+```shell
+npm install --save-dev @rushstack/pre-compile-hardlink-or-copy-plugin
```

Comment:
@iclanton Seems like such a simple and everyday problem should be solvable without having to load a plugin. I'm not even sure it should be a "plugin".  Heft should probably have a built-in facility for copying files around.  And also a system for invoking simple shell scripts before/after certain build events.

Is there a way we can decouple this feature from the Heft conversion?  Maybe just mark this package as non-publishing for now and delete the README?